src/nvim/api/private/helpers.c

```diff
@@ -1403,7 +1403,7 @@ bool set_mark(buf_T *buf, String name, Integer line, Integer col, Error *err)
       return res;
     }
   }
-  pos_T pos = { line, (int)col, (int)col };
+  pos_T pos = { (linenr_T)line, (int)col, (int)col };
```

Comment:
probably need to assert here, else coverity/PVS will complain

Reply:
assert of what exactly?

We could take care of the warnings after :)

Reply:
Meh, random asserts create a lot of warnings as well. I'll fix it.